store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -196,11 +235,26 @@ private CarbonTable buildCarbonTable() {
       tableSchemaBuilder = tableSchemaBuilder.blockletSize(blockletSize);
     }
 
-    List<String> sortColumnsList;
-    if (sortColumns != null) {
-      sortColumnsList = Arrays.asList(sortColumns);
+    if (isUnManagedTable) {
+      tableSchemaBuilder = tableSchemaBuilder.isUnmanagedTable(isUnManagedTable);
+    }
+
+    List<String> sortColumnsList = new ArrayList<>();
```

Comment:
Sort options behaviour should be consistent as create table 
If empty sort columns list, then it should be no sort.
If sort column list not passed, default behaviour should be same as create table.

Reply:
ok. done